src/CqlBuilderPanel/codesSection/CodesSection.tsx

```diff
@@ -69,6 +70,7 @@ export default function CodesSection({
           };
         });
         setParsedCodesList(codesList);
+        setSavedCodes(parsedCodesList ? parsedCodesList?.length : 0);
```

Comment:
we are using `parsedCodesList`, I wonder if we could get rid of line 42 and its usage

Reply:
@adongare I refined the code and used parsedCodesList instead of a new state variable, thanks Ashok!